alembic/versions/f16fdc26c6c6_add_shared_field_to_voicemails.py

```diff
@@ -0,0 +1,25 @@
+"""Add shared field to voicemails
+
+Revision ID: f16fdc26c6c6
+Revises: 22b03291f3ac
+
+"""
+
+import sqlalchemy as sa
+
+from alembic import op
+
+
+# revision identifiers, used by Alembic.
+revision = 'f16fdc26c6c6'
+down_revision = '22b03291f3ac'
+
+
+def upgrade():
+    op.add_column('voicemail', sa.Column('shared', sa.Boolean, nullable=False, server_default=sa.text('false')))
+    op.create_index('voicemail_shared_tenant_unique_key', 'voicemail', ['shared','context'], unique=True, postgresql_where=('shared'))
```

Comment:
```suggestion
    op.create_index('voicemail__idx__unique_shared_per_context', 'voicemail', ['shared','context'], unique=True, postgresql_where=(text("shared is true"))
```
From changes in xivo-dao

Reply:
Updated